packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -3572,6 +3572,366 @@ With the above two calls, we test the following configurations from this service
 
 Tests that we can grow up an operation from accepting one required parameter to accepting a required parameter and an optional parameter.
 
+### Routes_fixed
+
+- Endpoint: `get /routes/fixed`
+
+Simple operation at a fixed in an interface
+Expected path: /routes/fixed
+
+### Routes_InInterface
+
+- Endpoint: `get /routes/in-interface/fixed`
+
+Simple operation at a fixed in an interface
+Expected path: /routes/in-interface/fixed
+
+### Routes_PathParameters_annotationOnly
+
+- Endpoint: `get /routes/path/annotation-only`
+
+Path parameter annotated with @path but not defined explicitly in the route
+Value: "a"
+Expected path: /routes/path/annotation-only/a
+
+### Routes_PathParameters_explicit
+
+- Endpoint: `get /routes/path/explicit/{param}`
+
+Path parameter defined implicitly
+Value: "a"
+Expected path: /routes/path/explicit/a
+
+### Routes_PathParameters_Label_Explode_array
+
+- Endpoint: `get /routes/path/label/explode/array{.param*}`
+
+Test label expansion with explode: true when passed an array value.
+Param value: ["a","b"]
+Expected path: /routes/label/explode/array.a.b
+
+### Routes_PathParameters_Label_Explode_primitive
+
+- Endpoint: `get /routes/path/label/explode/primitive{.param*}`
+
+Test label expansion with explode: true when passed a primitive value.
```

Comment:
can we explode primitives? I thought explode had more to do with arrays and dicts

Reply:
The url template spec accept primitive types with explode, though it's ignoring explode since there is nothing to explode. So it's fair to have a test that check we don't "explode" (pun intended)